tardis/tests/test_atomic.py

```diff
@@ -4,8 +4,16 @@
 from tardis import atomic
 
 
-chianti_he_db_h5_path = os.path.join(
-    os.path.dirname(os.path.realpath(__file__)), 'data', 'chianti_he_db.h5')
+@pytest.fixture(scope="module")
+def default_atom_h5_path():
```

Comment:
I think It's fine calling `atomic.data_path` since you are testing that function, too.